inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -112,6 +114,43 @@ def get_flow_plane(self,
 
         return cutting_plane_mesh
 
+    def get_force_coefficients(self, save_path: Path = None):
+        """Get the force coefficients of the object in the WindTunnel.
+        
+        The force coefficients are provided in a .dat file during the
+        simulation run-time. This file contains 8 lines that are provide
+        the general input information. In this function, we read the file,
+        ignore the first 8 lines and read the force coefficients for the 
+        time_step chosen.
+
+        Args:
+            save_path: Path to save the force coefficients in a .csv file.
+        """
+
+        num_header_lines = 8
+        force_coefficients_path = os.path.join(self.sim_output_path,
+                                               "postProcessing", "forceCoeffs1",
+                                               "0", "forceCoeffs.dat")
+        force_coefficients = []
+
+        with open(force_coefficients_path, "r",
+                  encoding="utf-8") as forces_file:
+            for index, line in enumerate(forces_file.readlines()):
+                # Pick the line 8 of the file:
+                # [#, Time, Cm, Cd, Cl, Cl(f), Cl(r)]
+                if index == num_header_lines:
+                    force_coefficients.append(line.split()[1:])
+                # Add the force coefficients for the time_step chosen
+                elif index == num_header_lines + self.time_step + 1:
+                    force_coefficients.append(line.split())
```

Comment:
This is still a bit cryptic. Why are you indexing with `[1:]` the header line and not the others?

Reply:
I explain on top that is meant to drop the "#" entry on that list.